.asf.yaml

```diff
@@ -0,0 +1,8 @@
+github:
+  enabled_merge_buttons:
+    squash:  false
```

Comment:
If ever wants to allow merge directly from gh, can we enable squash? Because it is a step we do anyway when committing manually.

Reply:
Makes sense, I've updated the PR